pkg/controller/controller_utils.go

```diff
@@ -508,10 +508,10 @@ func validateControllerRef(controllerRef *metav1.OwnerReference) error {
 	if len(controllerRef.Kind) == 0 {
 		return fmt.Errorf("controllerRef has empty Kind")
 	}
-	if controllerRef.Controller == nil || *controllerRef.Controller != true {
```

Comment:
I don't care for `!*foo`.  Can you turn off that linting rule?

Reply:
Should we follow the rules of static checking？